### DISCUSSION TITLE
Companies comprised of a single person are exempt.

### DISCUSSION THREAD

**Discussion by milesgranger** [andrerav/regelverk-for-innleie#16] [General]
This is explicitly stated in the new regulation:

[Kjøp av tjenester fra en virksomhet som består av kun én person (enkeltpersonforetak eller selskap hvor eieren er eneste ansatt) vil heller ikke anses som innleie etter arbeidsmiljøloven.
](https://www.regjeringen.no/no/dokumenter/veileder-innleie-av-arbeidskraft/id2987562/?ch=1)

I've also had this confirmed by one of the biggest law firms in Norway as well as talking directly with a member of parliament. I've been seeing people talking about this affecting independent contractors, but I can't for the life of me understand why there is this confusion. 

---

Edit: I see this exemption is also mentioned here in the readme. It seems a reoccurring theme that people talking about this regulation glaze over this exemption then continue to talk about contract types or that single person companies won't be approved as staffing. All of which, in my view and apparently others, is irrelevant b/c they are exempt from the act in the first place.

**Comment by andrerav** [comment]
Hi @milesgranger. Yes, the law text does indeed have this statement. The problem is that this is being interpreted in different ways. 

As we have observed so far, customers appear to "err on the side of caution" and simply add requirements to their tenders that contractors must be employed at an approved staffing company -- without specifying that independent contractors are exempt from this requirement. Some customers are even going as far as explicitly stating that independent contractors are not allowed to be offered on tenders at all.

This interpretation is apparently referred to as a "misunderstanding" by government representatives when this question gets brought up in media and at "Stortingets spørretime". We will have to see what the new guidelines will say about this when they are published, and if they will clear up the misunderstandings.

**Comment by milesgranger** [comment]
> The problem is that this is being interpreted in different ways

I'm curious, what causes this you think? 

In my experience this is because people lead with something to the effect of "freelancing no longer available in Norway because....", then only later buried in the comments or in some off channels, others point out this exemption clause, and so the author of the original doom-and-gloom post responds with, "ya well there is some confusion here.".

The exemption statement, in my opinion, couldn't be any clearer. Unless of course the regulation led with it to put everyone at ease who doesn't care to read past the first paragraph; which seems to be a trend, at least on LinkedIn.

It feels like a similar story here, your own README leads with the statement of self-employed contractors will not get approval, but leaves buried the point of, oh right, they don't need to get approval because this whole thing doesn't apply to them anyway. 

This mess seems rather self inflicted, and it saddens me.

**Reply by andrerav** [reply]
That's a good point. While this primer does link to the law texts, it would be beneficial for the reader to have this specific part about the exemption quoted, emphasized and explained. I will try to improve the primer with regards to this. Thanks!
